portfolio/src/main/webapp/about.html

```diff
@@ -1,48 +1,55 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>About Me</title>
-     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
 
-  
+<head>
```

Comment:
Did you use the prettier formatter on these html files? Not sure why it increased the indentation spacing so much but it should be 2 spaces like before.